go.mod

```diff
@@ -73,7 +73,7 @@ require (
 	github.com/patrickmn/go-cache v2.1.0+incompatible
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.13.0
-	github.com/quasilyte/go-ruleguard/dsl v0.3.21
+	github.com/quasilyte/go-ruleguard/dsl v0.3.22
```

Comment:
did we miss to run `go mod tidy` in another PR?

Reply:
I think so - if I checkout `main` and run `make deps`, I get those changes every time.